app/src/main/res/layout/activity_option2.xml

```diff
@@ -42,4 +42,27 @@
         android:textAppearance="@style/text_title"
         android:background="@color/colorAccent"/>
 
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <Button
+            android:id="@+id/btn_next_2"
+            android:layout_width="wrap_content"
+            android:layout_height="@dimen/touch_target_size"
+            android:onClick="next"
```

Comment:
so if it's clicked which one is called? "next" as in XML or what in "setOnClickListener"?